lib/splunk/otel/instrumentation/action_pack.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Splunk
+  module Otel
+    module Instrumentation
+      # Contains the RUM instrumentation for the ActionPack gem
+      module ActionPack
```

Comment:
I think this file needs to be required from the root rb file, to be available in a regular Rails app.